sub-packages/bionemo-example_model/src/bionemo/example_model/training_scripts/finetune_mnist.py

```diff
@@ -0,0 +1,68 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: LicenseRef-Apache2
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import argparse
+from pathlib import Path
+
+from nemo.collections import llm
+from nemo.lightning import NeMoLogger, resume
+from pytorch_lightning.loggers import CSVLogger, TensorBoardLogger
+
+from bionemo.example_model.lightning.lightning_basic import (
+    BionemoLightningModule,
+    ExampleFineTuneConfig,
+    checkpoint_callback,
+    data_module,
+    trainer,
+)
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--pretrain_ckpt_dirpath", type=str, help="The checkpoint directory after pre-training")
+    args = parser.parse_args()
+
+    name = "example"
+    directory_name = "sample_models"
```

Comment:
are we going to run this as part of ci? i.e., through some test that executes it?
If so, it would be good to avoid dumping outputs in the repo root; you could have some configurable output directory.

If not, I worry that this could break without us knowing it -- unless you think we have enough coverage of this elsewhere